src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -79,6 +79,12 @@ def load_arguments(self, _):
     with self.argument_context('appconfig update') as c:
         c.argument('tags', arg_type=tags_type)
 
+    with self.argument_context('appconfig update', arg_group='Customer Managed Key', is_preview=True) as c:
+        c.argument('encryption_key_name', help='The name of the KeyVault key.')
+        c.argument('encryption_key_vault', help='The Uri of the KeyVault.')
```

Comment:
URI

Reply:
We are trying to keep consistent with ```az storage account update``` as below https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/storage/_params.py#L210 As cmk is kind of  new required features for all Azure services, we don't want to invert our own way.

Reply:
Sorry. I should make my advice clear. I mean usually URI is upper case.

Reply:
fixed.